packages/html/src/dialog/tests/dialog.tsx

```diff
@@ -50,7 +50,7 @@ export default () =>(
             </section>
 
             <section>
-                <DialogNormal title="Scrolling" actionButtonsAlign="end" style={{ height: "170px" }} >
+                <DialogNormal title="Scrolling" actionButtonsAlign="end" style={{ height: "220px" }} >
```

Comment:
I believe we should show scrollable content here and in Bootstrap this height removes the scrollable content. If we are increasing the height here, maybe we should add more content inside the dialog as well?